editoast/src/views/timetable/similar_trains.rs

```diff
@@ -1129,4 +1129,196 @@ mod tests {
             }
         );
     }
+
+    #[rstest]
+    async fn several_similar_trains() {
```

Comment:
```suggestion
    async fn prefer_single_train_over_compound_result() {
```

maybe more explanatory?

Reply:
Done.